contracts/aggregator/fantom/flashloan/main.sol

```diff
@@ -0,0 +1,183 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+/**
+ * @title Flashloan.
+ * @dev Flashloan aggregator for Fantom.
+ */
+import "./helpers.sol";
+import "hardhat/console.sol";
+
+contract FlashAggregatorFantom is Helper {
+    using SafeERC20 for IERC20;
+
+    event LogFlashloan(
+        address indexed account,
+        uint256 indexed route,
+        address[] tokens,
+        uint256[] amounts
+    );
+    
+    /**
+     * @dev Callback function for aave flashloan.
+     * @notice Callback function for aave flashloan.
+     * @param _assets list of asset addresses for flashloan.
+     * @param _amounts list of amounts for the corresponding assets for flashloan.
+     * @param _premiums list of premiums/fees for the corresponding addresses for flashloan.
+     * @param _initiator initiator address for flashloan.
+     * @param _data extra data passed.
+    */
+    function executeOperation(
+        address[] memory _assets,
+        uint256[] memory _amounts,
+        uint256[] memory _premiums,
+        address _initiator,
+        bytes memory _data
+    ) external returns (bool) {
+        bytes memory response_ = spell(AAVE_IMPL, msg.data);
+        return (abi.decode(response_, (bool)));
+    }
+
+    function routeFLA(
```

Comment:
Why does this route doesn't have a separate implementation, any particular reason?

Reply:
No reason as such, there was just a single function and no callback so implemented there itself. Tho mainnet won't have much space so adding this in a separate implementation.